src/patterns/asking-for-feedback/index.md.njk

```diff
@@ -0,0 +1,142 @@
+- who has access to it
+- where the people are who use it and the laws that apply there
+- what data is being collected by the third party (which is likely to be more than the questions we add to their tool)
+- how data may be used by a third party
+
+## Keep people’s data secure
+
+Government has a legal duty to protect user information. If we fail in that duty, we will undermine public trust in government services.
+
+The [Data Protection Act 2018](https://www.legislation.gov.uk/ukpga/2018/12/contents/enacted) controls how we use personal information in government.
+
+In your feedback form, you must [consider what information you collect from your users](https://www.gov.uk/service-manual/design/collecting-personal-information-from-users).
+
+You should always collect information anonymously when users are signed in.
+
+You must avoid collecting personal information about users if you do not intend to. For example, you may accidentally collect their name or email address if they are signed into a staff network.
+
+Where it would be useful to know who someone is, or to collect other personally identifying information you need to be clear about:
+
+- why you’re collecting it
+- what you’re collecting
+- how you'll use it
+- who has access to it
+- how long we’ll keep it
+- that it will be secure
+- how they can find out more or raise a concern or complaint
+
+[Find out more about the Data Protection Act and what it covers](https://www.gov.uk/data-protection)
+
+## Welsh language services
+
+Services in Welsh should provide feedback forms in Welsh.
+
+Check our [HM Land Registry Welsh Language Scheme](https://www.gov.uk/government/organisations/land-registry/about/welsh-language-scheme) if you are not sure if this applies to your service.
+
+If your service is being used by Welsh speaking people, you should do this as soon as possible. By public beta all parts of the service should be in Welsh including the feedback form.
+
+Follow our guidance for help with developing a Welsh version of your form.
+
+## Related links
+
+- [Get a feedback page on GOV.UK](https://www.gov.uk/service-manual/service-assessments/get-feedback-page)
+- [How to structure forms](https://www.gov.uk/service-manual/design/form-structure)
+
+## Why you should follow this pattern advice
+
+It’s also part of your service under the law. For example, in the Equality Act 2010 we fall under the 'service provider' category.
+
+The rest of the team should consider this page as they would any other in the services, for example:
+
+- lawyers confirming compliance with all laws
+- testers ensuring its technical accessibility
+- software developers should ensure, for example it’s secure, is available, what happens if the service (or part of) is not available.
+- user researchers and designers to ensure its usable and accessible (non-technical compliance
+
+The feedback form must be accessibility tested as part of the accessibility statement.
+
+## Accessibility and inclusivity
```

Comment:
Is this somewhat a duplication of what is in the Accessibility section of the Design System?